opds.py

```diff
@@ -222,6 +218,18 @@ def authors(cls, work, license_pool, edition, identifier):
         return [E.author(E.name(edition.author or ""))]
 
     @classmethod
+    def series(cls, series_name, series_position):
+        """Generate a schema:Series tag for the given name and position."""
+        if not series_name:
+            return None
+        series_details = dict()
+        series_details['name'] = series_name
+        if series_position:
+            series_details['{%s}position' % schema_ns] = unicode(series_position)
```

Comment:
The element maker expects a string. Trying to pass an integer raises an error. 😞